static/css/page.css

```diff
@@ -68,6 +68,6 @@
 }
 
 .right {
-	float: right;
 	margin-bottom: 20px;
+	text-align: center;
```

Comment:
I feel like this change might be confusing, given the name of the class. The class is only used in 2 places - rename?

Reply:
Seems to be **very** plausible to change the class name. :-)